src/content/configuration/stats.md

```diff
@@ -320,7 +320,7 @@ module.exports = {
 
 `boolean = false`
 
-Tells `stats` whether to hide `orphan` modules. A module is an `orphan` if it is not included in any chunk. Orphans modules are now hidden by default in `Stats`.
+Tells `stats` whether to hide `orphan` modules. A module is an `orphan` if it is not included in any chunk. Orphan modules are hidden by default in `stats`.
```

Comment:
```suggestion
Tells `stats` whether to hide `orphan` modules. A module is `orphan` if it is not included in any chunk. Orphan modules are hidden by default in `stats`.
```

Reply:
Should we mention , from webpack 5

Now could be replaced by webpack 5 ig

Reply:
I think `an` is good, just ran it through my native speaker mate.

@jeffin143 It says webpack 5 docs in the left hand side, so we should stop doing these mentions probably